st2common/st2common/models/system/action.py

```diff
@@ -86,6 +88,18 @@ def get_timeout(self):
     def get_cwd(self):
         return self.cwd
 
+    def _get_env_vars_export_string(self):
+        if self.env_vars:
+            exports = ' '.join(
+                '%s=%s' % (quote_unix(k), quote_unix(v))
```

Comment:
Then please add a comment saying why it is needed. Else, it could easily be dropped in some refactoring/clean-up effort.